packaging/windows_wix/package_info.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import print_function
+from os import path
+import toml
+
+
+def get_version():
+    with open(path.join("..", "..", "Cargo.toml")) as f:
+        cargo = toml.load(f)
+        return cargo["package"]["version"]
```

Comment:
I think this should be like this:

```python
def get_version():
    with open(path.join("..", "..", "Cargo.toml")) as f:
        cargo = toml.load(f)
    return cargo["package"]["version"]
```

Then the context is closed explicitly before returning the value.